ada_2_fhir-r4/cio/2.0.0/beschikbaarstellen_contraindicatie/fhir_instance/pracrole-000003332.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<PractitionerRole xmlns="http://hl7.org/fhir">
+   <id value="pracrole-000003332"/>
+   <meta>
+      <profile value="http://nictiz.nl/fhir/StructureDefinition/nl-core-HealthProfessional-PractitionerRole"/>
+   </meta>
+   <practitioner>
```

Comment:
Algemene vraag, misschien vreemde locatie, maar waarom zie ik geen practitioner resource terug? Ik kan alleen practionerrole resources vinden

Reply:
Goed gespot! Dit moet ik nog oplossen (ik ga kijken of dat vandaag lukt, anders voor de volgende versie).